src/main/scala/vectorpipe/VectorPipe.scala

```diff
@@ -151,7 +155,34 @@ object VectorPipe {
       saveVectorTiles(vts, zoom, pipeline.baseOutputURI)
       prepared.withColumn(keyColumn, reduceKeys(col(keyColumn)))
     }
-
   }
 
+  /** Construct a SparkSession with defaults tailored for use with VectorPipe
+   *  This session enables:
+   *   - Hive support
+   *   - JTS support
+   *   - Kryo serialization
+   *   - spark master local[*] if unset for local testing
+   *
+   *  If you need more control over config, you'll need to construct your own session
+   */
+  def defaultSparkSessionWithJTS(appName: String): SparkSession = {
+    val conf = new SparkConf()
+      .setIfMissing("spark.master", "local[*]")
+      .setAppName(appName)
+      .set("spark.sql.orc.impl", "native")
+      .set("spark.sql.orc.filterPushdown", "true")
+      .set("spark.sql.parquet.mergeSchema", "false")
+      .set("spark.sql.parquet.filterPushdown", "true")
+      .set("spark.sql.hive.metastorePartitionPruning", "true")
+      .set("spark.ui.showConsoleProgress", "true")
+      .set("spark.serializer", classOf[KryoSerializer].getName)
+      .set("spark.kryo.registrator", classOf[KryoRegistrator].getName)
+
+    SparkSession.builder
+      .config(conf)
+      .enableHiveSupport
+      .getOrCreate
+      .withJTS
+  }
```

Comment:
Ooo.  Nice.  Me likey.

Reply:
Whew.

I'll be curious to see if this gets used anywhere else.